plugins/command-not-found/command-not-found.plugin.zsh

```diff
@@ -1,8 +1,22 @@
 # Uses the command-not-found package zsh support
 # as seen in https://www.porcheron.info/command-not-found-for-zsh/
 # this is installed in Ubuntu
-
-[[ -e /etc/zsh_command_not_found ]] && source /etc/zsh_command_not_found
+if [ -x /usr/lib/command-not-found -o -x /usr/share/command-not-found/command-not-found ]; then
+	command_not_found_handler() {
+        # check because c-n-f could've been removed in the meantime
+        if [ -x /usr/lib/command-not-found ]; then
+            /usr/lib/command-not-found -- "$1"
+            return $?
+        elif [ -x /usr/share/command-not-found/command-not-found ]; then
+            /usr/share/command-not-found/command-not-found -- "$1"
+            return $?
+        else
+            printf "%s: command not found\n" "$1" >&2
```

Comment:
This should be `zsh: command not found: %s\n` to match what zsh prints by default. Also there are tabs mixed with spaces, can you convert all tabs to spaces?